ui/store/actions.ts

```diff
@@ -4365,11 +4366,14 @@ export function setTransactionSecurityCheckEnabled(
     }
   };
 }
+///: END:ONLY_INCLUDE_IN
 
 ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
 export function setSecurityAlertsEnabled(val: boolean): void {
   try {
     submitRequestToBackground('setSecurityAlertsEnabled', [val]);
+    // also update the old settings
+    submitRequestToBackground('setTransactionSecurityCheckEnabled', [val]);
```

Comment:
Same as before. Why are we setting `setTransactionSecurityCheckEnabled`?

Reply:
yes, because of the requirement to enable blockaid if old security check is enabled.

Reply:
humm.. but this action is fired when a user explicit wants to enable the new blockaid security check, right? So why are we also setting the old property? That shouldn't be needed anymore right? 🤔

Reply:
it could also be fired when user is turning off blockaid. But like Jyoti commented, doing a migration is better.